package.json

```diff
@@ -23,11 +23,12 @@
   },
   "homepage": "https://github.com/fastify/fastify-sensible#readme",
   "devDependencies": {
-    "fastify": "^2.0.0-rc.2",
+    "fastify": "^2.0.0",
```

Comment:
update fastify to latest v2 release

Reply:
This is actually from https://github.com/fastify/fastify-sensible/commit/b8985e8005300ed6ad17ac81a9b51c92315a232e. Should I update to use 2.4.1 (latest from @fastify)?

Reply:
Hi @Ethan-Arrowood , can you confirm request regarding this change? Thanks

Reply:
Yeah I don't see why not